<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookMarketManageWidget</class>
 <widget class="QWidget" name="BookMarketManageWidget">
  <property name="windowTitle">
   <string>Book Market</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="pixmap">
        <pixmap resource="res.qrc">:/pic/title.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="openCloseBtn">
       <property name="text">
        <string>Closed</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/pic/shut_down_grayed.png</normaloff>:/pic/shut_down_grayed.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="opList">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>58</width>
          <height>58</height>
         </size>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <property name="currentRow">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Borrow</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/pic/borrow.png</normaloff>:/pic/borrow.png</iconset>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Back</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/pic/back.png</normaloff>:/pic/back.png</iconset>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manage</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/pic/manage.png</normaloff>:/pic/manage.png</iconset>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="PageBorrowWidget" name="borrowPage"/>
        <widget class="PageBackWidget" name="backPage"/>
        <widget class="PageManageWidget" name="managePage"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PageBorrowWidget</class>
   <extends>QWidget</extends>
   <header>PageBorrowWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageBackWidget</class>
   <extends>QWidget</extends>
   <header>PageBackWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageManageWidget</class>
   <extends>QWidget</extends>
   <header>PageManageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>opList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
